packages/rum-core/src/common/config-service.js

```diff
@@ -180,7 +179,7 @@ class Config {
       properties.serverUrl = properties.serverUrl.replace(/\/+$/, '')
     }
 
-    this.config = merge({}, this.defaults, this.config, properties)
+    this.config = merge({}, this.config, properties)
```

Comment:
I don't think we still to pass a new object here, we can simply merge into `this.config`